subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -490,22 +490,52 @@ As the way to declare the repositories and what they are expected to contain dep
 [[sub:centralized-repository-declaration]]
 == Centralizing repositories declaration
 
-Instead of declaring repositories in every subproject of your build or via an `allprojects` block, Gradle offers a way to declare them in a central place for all project.
+Instead of declaring repositories in every subproject of your build or via an `allprojects` block, Gradle offers a way to declare them in a central place for all projects.
 
-NOTE: Central declaration of repositories is an incubating feature
+NOTE: Central declaration of repositories is an incubating feature.
 
-Repositories used by convention by every subproject can be declared in the `settings.gradle(.kts)` file:
+Repositories used by convention in every subproject can be declared in the `settings.gradle(.kts)` file:
 
 .Declaring a Maven repository in settings
 ====
 include::sample[dir="snippets/artifacts/defineRepositoryInSettings/kotlin",files="settings.gradle.kts[tags=declare_repositories_settings]"]
 include::sample[dir="snippets/artifacts/defineRepositoryInSettings/groovy",files="settings.gradle[tags=declare_repositories_settings]"]
 ====
 
-The `dependencyResolutionManagement` repositories block accepts the same notations as in a project, which includes Maven or Ivy repositories, with or without credentials, etc.
+The `dependencyResolutionManagement` repositories block accepts the same notations as in a project. This includes Maven or Ivy repositories, with or without credentials, etc.
 
 By default, repositories declared by a project will *override* whatever is declared in settings.
-You can change this behavior to make sure that you always use the settings repositories:
+This means that repositories declared in a project `build.gradle(.kts)` file *override* those declared in the `settings.gradle(.kts)` file:
+
+.Preferring project repositories
+====
+include::sample[dir="snippets/artifacts/defineRepositoryInSettings/kotlin",files="settings.gradle.kts[tags=prefer_projects]"]
+include::sample[dir="snippets/artifacts/defineRepositoryInSettings/groovy",files="settings.gradle[tags=prefer_projects]"]
+====
+
+There are three modes for dependency resolution management:
+
+[cols="1,1,1,1"]
+|===
+|Mode |Description |Default? |Use-Case
+
+|`PREFER_PROJECT`
+|Any repository declared on a project will cause the project to use the repositories declared by the project, ignoring those declared in settings.
+|Yes
+|Useful when teams need to use many different plugins not shared among subprojects.
+
+|`PREFER_SETTINGS`
+|Any repository declared directly in a project, either directly or via a plugin, will be ignored.
+|No
+|Useful for enforcing large teams to use the same plugin versions.
```

Comment:
I would expect the use case here to be the same as `FAIL_ON_PROJECT_REPOS` just that this option wouldn't fail the build.

```suggestion
|Useful for enforcing large teams to use approved repositories only, but will not fail the build when a project or plugin declares a repository.
```